kuma/javascript/src/l10n.test.jsx

```diff
@@ -152,3 +160,28 @@ describe('interpolate()', () => {
         );
     });
 });
+
+describe('<Interpolated/>', () => {
+    it('takes element parameters', () => {
+        const tree1 = renderer
+            .create(
+                <Interpolated
+                    id="oh a <link/> wow"
+                    link={<a href="/">click</a>}
+                />
+            )
+            .toJSON();
+        expect(tree1).toMatchSnapshot();
+
+        const tree2 = renderer
+            .create(
+                <Interpolated
+                    id="wow <head/> such <body /> many <foot />"
+                    head={<h1>Beautiful</h1>}
+                    foot={<p className="something">more</p>}
+                />
+            )
+            .toJSON();
+        expect(tree2).toMatchSnapshot();
```

Comment:
Perhaps we can add another test where just a string is passed and we can then confirm that the string is returned unchanged?